experiment/test_run_experiment.py

```diff
@@ -199,19 +199,30 @@ def test_copy_resources_to_bucket(tmp_path):
     config_dir = 'config'
     config = {
         'experiment_filestore': 'gs://gsutil-bucket',
-        'experiment': 'experiment'
+        'experiment': 'experiment',
+        'benchmarks': ['libxslt_xpath'],
+        'oss_fuzz_corpus': True,
     }
     try:
-        with mock.patch('common.filestore_utils.rsync') as mocked_rsync:
-            with mock.patch('common.filestore_utils.cp') as mocked_cp:
-                run_experiment.copy_resources_to_bucket(config_dir, config)
-                mocked_cp.assert_called_once_with(
-                    'src.tar.gz',
-                    'gs://gsutil-bucket/experiment/input/',
-                    parallel=True)
-                mocked_rsync.assert_called_once_with(
-                    'config',
-                    'gs://gsutil-bucket/experiment/input/config',
-                    parallel=True)
+        with mock.patch('common.filestore_utils.cp') as mocked_filestore_cp:
```

Comment:
It's surprising to me that you want to mock both the filestore and gsutil versions.